manticore/platforms/evm.py

```diff
@@ -469,6 +470,33 @@ def __init__(self, constraints, address, data, caller, value, bytecode, world=No
             bytecode_symbolic[0:bytecode_size] = bytecode
             bytecode = bytecode_symbolic
 
+        #TODO: Handle the case in which bytecode is  symbolic (This happens at
+        # CREATE instructions that has the arguments appended to the bytecode)
+        self._check_jumpdest = False
+        self._valid_jumpdests = set()
+
+        #Compile the list of valid jumpdests via linear dissassembly
+        def extend_with_zeroes(b):
+            # Fixme(felipe) This does not deal with the corner case in which the
+            # arguments are actually dynamic code 
+            # This simplifies the bytecode that ends early or has symbolic arguments
+            # appended at the end
+            try:
+                for x in b:
+                    x = to_constant(x)
+                    if isinstance(x, int):
+                        yield(x)
+                    else:
+                        yield(0)
+                for _ in range(32):
+                    yield(0)
```

Comment:
I am not that much into the topic, but we add 32 * 0 no matter of the bytecode length, shouldn't we do sth like `32 - len(bytecode)` or something similar? Just sanity check/question huh

Reply:
So to sum up, do I understand correctly that in most cases we need to add only *some* zeros but in the worst case all of 32 zeros. And the fact we add 32 zeros always is okay i.e. it doesn't change any other behavior?

Reply:
So this linear disassembly thing is only used to search for "JUMPDEST" instructions and whitelisting those as JUMP targets. Adding zeros (STOP opcode or zero padded PUSH operand) will not add new valid JUMPDEST targets.